weather_dl/download_pipeline/pipeline_test.py

```diff
@@ -632,5 +635,19 @@ def test_target_name_date_and_target_directory_additional_partitions_in_path(sel
         self.assertEqual(target_name, "somewhere/expver-1/2017/01/15-pressure-500.nc")
 
 
+class ConfigTest(unittest.TestCase):
+
+    def setUp(self):
+        self._data_dir = f'{next(iter(weather_dl.__path__))}/../configs'
+
+    def test_process_config_files(self):
```

Comment:
Ideally, I would like to see a separate test failure message for each config that fails. In other words, it would be nice to generate separate tests methods for each config in the directory. 

The simplest is to use unittest's [sub tests](https://docs.python.org/3/library/unittest.html#distinguishing-test-iterations-using-subtests).

Reply:
Thanks for the tip! 

Note this required installation of [pytest-subtests](https://pypi.org/project/pytest-subtests/)